usort/tests/sort_key.py

```diff
@@ -89,9 +89,30 @@ def test_from_node_ImportFrom_relative(self) -> None:
 
 
 class IsSortableTest(unittest.TestCase):
+    def test_is_side_effect(self) -> None:
+        config = Config(side_effect_modules=["fizzbuzz", "foo.bar.baz"])
```

Comment:
I had to stare at this for a bit to understand.  Can you split the fizzbuzz and foo.bar.baz into separate tests, and also add one that simple string prefix matches are false (e.g. fizzbuzz1 for x.fizzbuzz does not match fizzbuzz)